example/README.md

```diff
@@ -1,16 +1,15 @@
 # example
 
-A new Flutter project.
+This example project shows some basic usages of the `tinycolor2` package. 
 
-## Getting Started
+## Run project
 
-This project is a starting point for a Flutter application.
+### In your favourite  IDE
+Open `tinycolor2/example` in your favourite IDE and run it from there. While `tinycolor2`
```

Comment:
I think null safety has merged #24.

Reply:
@calvintam236 ok, that went completely unnoticed by me